### PR TITLE
Add make-install-var field to the make plugin.

### PR DESCRIPTION
**Pull request by blakerouse**
Each project is different on the install flag for the Makefile. Allow changing the make-install-var when building with snapcraft.

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by sergiusens** [comment]
looks good, ok to test

**Comment by sergiusens** [comment]
thanks btw!

File: snapcraft/tests/test_plugin_make.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ def setUp(self):
         class Options:
             makefile = None
             make_parameters = []
+            make_install_var = "DESTDIR"
```
Comment (kyrofa) [inline snapcraft/tests/test_plugin_make.py]: Single quotes here, please.

File: snapcraft/plugins/make.py [hunk 2]
```diff
@@ -32,6 +32,10 @@
     - make-parameters:
       (list of strings)
       Pass the given parameters to the make command.
+
+    - make-install-var:
+      (string; default: DESTDIR)
+      Use the variable as the install target.
```
Comment (kyrofa) [inline snapcraft/plugins/make.py]: This is a little confusing-- "install" **is** the target. Perhaps something like "Use this variable to redirect the installation into the snap" or something like that.

**Comment by blakerouse** [comment]
Fixed the requested changes.

**Pull request merged**